js/main.js

```diff
@@ -28,18 +28,18 @@ $form.addEventListener('submit', function (event) {
   $entryformDiv.className = 'hidden';
   $entriesDiv.className = 'entries';
   data.view = 'entries';
+  $ulElement.prepend(treeMaker(entryObj));
   $form.reset();
-  window.location.reload();
 });
 
 function treeMaker(entry) {
 
-  var ulElement = document.createElement('ul');
-  ulElement.setAttribute('class', 'entry-list');
+  var container = document.createElement('div');
+  container.setAttribute('class', 'container-column');
 
   var liImg = document.createElement('li');
   liImg.setAttribute('class', 'entry-item');
-  ulElement.appendChild(liImg);
+  container.appendChild(liImg);
```

Comment:
You will want to reverse these. `li` elements should be the only type of element that is a child to a `ul`. You can have the `div` directly inside the `li`